### ISSUE TITLE
Update contributing instructions

### ISSUE THREAD

**Issue by BenSpencer** [naquiroz/heroicons-lookup#3]
Hey, I wanted to update the package to include some new icons (or their new names), which I have done in https://github.com/naquiroz/heroicons-lookup/pull/2

But I found the contributing instructions hard to follow, lots of the info in the template doesn't seem to match this project. I wasn't sure what direction you might want to switch (e.g. eslint vs prettier vs whatever) so I haven't made any changes directly.

The parts I spotted:
- Are the npm/node versions right? I had npm 6.x on my machine and this rewrote the entirety of package-lock.json to use lockfileVersion:1. Switching to the latest (9.x) resolved it, and I didn't dig into exactlly what version is expected.
- In step 3 it says `npm` where I assume it means `npm install`?
- Step 4 doesn't seem relevant at all
- I wasn't clear if the tip was actually wanted?
- Under the PR submission instructions it says to run the tests, but there are no tests.
- It also says to run `npm run lint` but that command doesn't work. It's defined as `eslint src` but needs to be something like `eslint src/**/*.ts`. But switching to that just makes eslint complain that there's no config file.
- I could run prettier OK with `npx prettier -w src` in case that's useful to include.

**Comment by naquiroz** [comment]
Thanks for the suggestions! Yes, I was in a bit of a hurry when I did the contribution guide, I'll have a look at it and edit it. Would you mind reviewing it after I make a PR?

**Comment by BenSpencer** [comment]
Happily! I can also help, I just wasn't sure which direction you'd prefer so I didn't just make up something entirely different myself 🙃

**Comment by naquiroz** [comment]
Sorry about this, do you still need me to push it? I plan to put this package back on track

**Comment by BenSpencer** [comment]
I'm still happy to review if useful, but it's not blocking for the work I was doing. 👍

**Comment by naquiroz** [comment]
Hi, apologies for the very long wait. V2 released!

**Comment by naquiroz** [comment]
Documentation needs rework, feel free to contribute
